src/session-pool.ts

```diff
@@ -302,21 +300,10 @@ const enum errors {
 }
 
 interface SessionInventory {
-  [types.ReadOnly]: Session[];
-  [types.ReadWrite]: Session[];
+  sessions: Session[];
   borrowed: Set<Session>;
 }
 
-interface Waiters {
-  [types.ReadOnly]: number;
-  [types.ReadWrite]: number;
-}
-
-export interface CreateSessionsOptions {
```

Comment:
Same here: Removing this interface all together is a breaking change.

Reply:
Reverted back.